src/build/mod.rs

```diff
@@ -68,7 +68,7 @@ pub fn build_target(target: &Target) -> Result<String> {
     }
 }
 
-pub fn command(args: &[&str], binary_path: &PathBuf) -> Command {
+pub fn command(args: &[&str], binary_path: &Path) -> Command {
```

Comment:
What's the purpose of changing `PathBuf` to `Path`?

Reply:
It's no less flexible, and it makes it a little simpler because you're no longer going through Deref every time.

```
warning: writing `&PathBuf` instead of `&Path` involves a new object where a slice will do
   --> src/wranglerjs/mod.rs:227:25
    |
227 | fn run_npm_install(dir: &PathBuf) -> Result<()> {
    |                         ^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg
```

Reply:
👍